src/parser.rs

```diff
@@ -46,7 +46,7 @@ pub(crate) fn parse_params(input: &str) -> IResult<&str, Option<LocationParamete
                     LocationParamKeys::Submodules => params.set_submodules(Some(value.into())),
                     LocationParamKeys::Shallow => params.set_shallow(Some(value.into())),
                     LocationParamKeys::Arbitrary(param) => {
-                        params.add_arbitrary((param, value.into()))
+                        params.add_arbitrary((param, value.into()));
```

Comment:
What lint is that? I can't reproduce that with:

```
cargo clippy
```

Reply:
My bad, should have edited the PR comments. Basically this:

https://github.com/Ben-PH/nix-uri/blob/b702d95b3ff724298a63d2f21dc6fd59ad8f38eb/src/lib.rs#L1-L9

I've put one clippy warning per line, and added some `allow`s to take out the noise.

The idea being, is that by default, it's commented out. The normal dev process isn't affected by them, but every now and then, you can uncomment them, and go through things manually. 

If you run `cargo clippy -- -W clippy::pedantic, that will trigger this lint, I believe. alternatively, uncomment the warn-lines, and you should see the lint triggered.